activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +37,16 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
+      # the output configured by <tt>ActiveSupport::Deprecation.behavior</tt>.
+      # By default +caller_backtrace+ is equal to caller(2).
+      #
+      #   ActiveSupport::Deprecation.deprecation_warning('something broke!')
```

Comment:
What do you think if i use `'deprecated method name'` instead `'something broke!'`?

So, in `warn` output and return values is equal, but in `deprecation_warning` this values is different. For me output value is more important because this value you see in log. And therefore I indicate output value in `=>`